torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -61,6 +60,7 @@ void PythonRpcHandler::cleanup() {
   pySerialize_ = py::none();
   pyHandleException_ = py::none();
   jitCompilationUnit_ = nullptr;
+  typeParser_ = nullptr;
```

Comment:
Curious, it does not seem `ScriptTypeParser` depends on Python. Is setting it to nullptr here just being defensive or will it indeed hit error in certain destruction order?